### DISCUSSION TITLE
Q-key quits game without warning in Main Menu

### DISCUSSION THREAD

**Discussion by MaxKruse** [ppy/osu#23427] [Q&A]
In stable, using the Q-key in main menu to quit forces a popout for confirmation. In lazer, it simply expands the main menu on the first click, and quits the game on the second. Might become confusing for players switching over. Not a dealbreaker or anything, but definitly weird that there isnt a confirmation to close.

**Comment by peppy** [comment]
I can't find the tracking issue, but the problem here is that the confirmation dialog shows in all cases, rather than only when there's a reason to require confirmation.
